### ISSUE TITLE
Homebridge Verified?

### ISSUE THREAD

**Issue by shanelord01** [danTapps/homebridge-hubitat-makerapi#11]
Hi,

This plugin is awesome and helps my Hubitat/Homebridge world work.

Any chance it could be updated to be verified?
https://github.com/homebridge/homebridge/wiki/verified-Plugins

Thanks,
Shane.

**Comment by AppleTechy** [comment]
My pending PR adds the Config.Schema.json that is part of what is required to make it a verified plugin.

**Comment by cmfrazier** [comment]
Any update on this PR?

**Comment by AppleTechy** [comment]
> Any update on this PR?

Nope. Still waiting for plugin maintainer to take a look at it. It's still pending.
